standalone-node/docs/user-guide/desktop-virtualization-cloud-init.md

```diff
@@ -67,9 +82,15 @@ write_files:
 #   runcmd:
 #     - systemctl restart myservice
 #     - bash /etc/cloud/test.sh
+# If custom scripts in user-apps are getting invoked. please make sure to add the scripts in "user-apps" folder
+# Example:
+#   runcmd:
+#     - bash /opt/user-apps/network-config.sh /etc/cloud/custom_network.conf
 runcmd:
   - echo $(( 6 * 1024 * 4 )) | sudo tee /sys/kernel/mm/hugepages/hugepages-2048kB/nr_hugepages
   - systemctl --user enable idv-init.service
   - udevadm control --reload-rules
+  - bash /opt/user-apps/network_config.sh /etc/cloud/custom_network.conf
```

Comment:
/opt/user-apps/network_config.sh or the config file is missing/executable flag is not set, this will fail.